### PR TITLE
[plugin.video.catchuptvandmore] 0.1.2

### PR DESCRIPTION
**Pull request by SylvainCecchetto**
### Description
Update of Catch-up TV & More plugin from v0.1.0 to v0.1.2

**Changelog :**
 
[Add channel] 6ter
[Bug] NRJ 12: No video
[Improvement] NRJ 12: Add listing for categories and seasons
[Improvement] France Télévision (pluzz): Add search feature and "last videos" categories
[Bug] TF1: Video URL failed
[Bug] Gulli: Doesn't work
[Add channel] Numéro 23
[Add channel] Chérie 25
[Bug] Arte: Doesn't work
[Bug] Canal+: Bug with subcategories
[Add channel] France Ô
[Add channel] HD1
[Add channel] L'Équipe

Thanks a lot for your job!

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.catchuptvandmore/addon.xml [hunk 1]
```diff
@@ -1,32 +1,42 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="plugin.video.catchuptvandmore"
        name="Catch-up TV &amp; More"
-       version="0.1.0"
+       version="0.1.2"
        provider-name="SylvainCecchetto">
   <requires>
-    <import addon="xbmc.python" version="2.25.0"/>
+    <import addon="xbmc.python" version="2.24.0"/>
```
Comment (enen92) [inline plugin.video.catchuptvandmore/addon.xml]: You are targetting Krypton so you don't need simplejson. The python api version was correct as well.

File: plugin.video.catchuptvandmore/addon.xml [hunk 2]
```diff
@@ -1,32 +1,42 @@
...
-    <summary lang="fr_fr">Catch-Up TV &amp; More : Replay TV et bien plus encore.</summary>
-    <description lang="fr_fr">Catch-Up TV &amp; More regroupe dans un même plugin Kodi l'ensemble des vidéos des différents services et chaînes de Replay TV (rattrapage TV). De plus, cette extension vous permet d'accéder rapidement aux vidéos et contenus proposés par certains sites internet.</description>
-    <disclaimer lang="fr_fr">Retours de bugs, propositions d'améliorations ou d'ajout de contenus sont les bienvenue ! GitHub ou e-mail.</disclaimer>
-    <summary lang="en_gb">Catch-Up TV &amp; More: It's all in the title.</summary>
-    <description lang="en_gb">Catch-Up TV &amp; More brings together in one Kodi plugin all the videos of the various services and channels of catch-up TV. Furthermore, this addon allows you to quickly access the videos and content offered by certain websites.</description>
-    <disclaimer lang="en_gb">Bug reports, suggestions for improvements or content additions are welcome! GitHub or e-mail.</disclaimer>
-    <language>fr</language>
+    <summary lang="fr">Catch-Up TV &amp; More : Replay TV et bien plus encore.</summary>
```
Comment (enen92) [inline plugin.video.catchuptvandmore/addon.xml]: This was also fine. I think you reverted quite a lot of stuff that was correct with this PR

**Comment by SylvainCecchetto** [comment]
Sorry it's a mistake. Thanks a lot for your review.
I just pushed a new commit without the previous mistakes and, I think, only the new features.

Thanks!

File: plugin.video.catchuptvandmore/resources/lib/simpleplugin.py [hunk 3]
```diff
@@ -504,7 +504,7 @@ def initialize_gettext(self):
         :return: :meth:`Addon.gettext` method object
         :raises simpleplugin.SimplePluginError: if the addon's English :file:`strings.po` file is missing
         """
-        strings_po = os.path.join(self.path, 'resources', 'language', 'resource.language.en_gb', 'strings.po')
+        strings_po = os.path.join(self.path, 'resources', 'language', 'English', 'strings.po')
         if os.path.exists(strings_po):
```
Comment (enen92) [inline plugin.video.catchuptvandmore/resources/lib/simpleplugin.py]: I doubt you're allowed to read the strings file directly / parse it / change it from your plugin. Why do you need this? The API has methods to retrieve the strings you need.

Comment (enen92) [inline plugin.video.catchuptvandmore/resources/lib/simpleplugin.py]: I'd drop the class all together. Please fix it so it can be merged.

**Comment by SylvainCecchetto** [comment]
Hi! Sorry for the time... I thinks that it's ok now? This "simpleplugin.py" file come from https://github.com/romanvm/script.module.simpleplugin/.
The plugin now work with the correct strings.po path.
Thanks a lot for your work!

**Pull request merged**